MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -3240,7 +3378,7 @@ public string GetProgressiveVideoArguments(EncodingJobInfo state, EncodingOption
 
                 if (state.RunTimeTicks.HasValue && state.BaseRequest.CopyTimestamps)
                 {
-                    args += " -copyts -avoid_negative_ts disabled -start_at_zero";
+                    args += " -copyts -avoid_negative_ts 0 -start_at_zero";
```

Comment:
Why this change? The docs strongly recommend using the named values instead of the ints.

Reply:
I forgot to change it back here, I wanted to understand how it works.